Classes/Error/GraylogExceptionHandler.php

```diff
@@ -28,10 +28,15 @@ class GraylogExceptionHandler extends ProductionExceptionHandler
      */
     protected function echoExceptionWeb($exception)
     {
+
         if ($this->graylogService === null) {
             $this->graylogService = new GraylogService();
```

Comment:
&lt;nitpick&gt;we only need to instantiate the `graylogService` if `logException` is true so this could be moved in the if statement below